x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/analytics_list.tsx

```diff
@@ -284,11 +286,11 @@ export const DataFrameAnalyticsList: FC<Props> = ({
           hasActions={false}
           isExpandable={true}
           isSelectable={false}
-          items={pageOfItems}
+          items={pageOfItems as DataFrameAnalyticsListRow[]}
```

Comment:
with generics and overloads, you can avoid a manual typecasting

Reply:
Good call - updated types in e132747913497159f245f33bca035eb9e2893cac